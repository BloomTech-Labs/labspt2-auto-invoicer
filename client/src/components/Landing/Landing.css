.landing {
  display: flex;
  flex-direction: column;
  width: 100%;
  align-items: center;
}

@media (max-width: 500px) {
  .button {
    font-size: 1rem;
    width: 100%;
    padding: 2px 0;
  }
}
