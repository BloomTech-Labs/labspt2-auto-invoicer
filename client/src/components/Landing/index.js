import React from "react";

//import LandingText from "../LandingText";
import LandingCTA from "../LandingText/LandingCTA.js";
import LandingFeaturesText from "../LandingText/LandingFeaturesText";
import { LandingGetStarted } from "../LandingGetStarted";

import HeadlineText from "../LandingText/HeadlineText";

const Landing = props => {
  return (
    <React.Fragment>
      <HeadlineText />
      <LandingCTA />
      <LandingFeaturesText />

      <LandingGetStarted />
    </React.Fragment>
  );
};

export default Landing;
