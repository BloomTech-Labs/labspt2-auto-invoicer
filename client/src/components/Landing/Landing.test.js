import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import Landing from "../Landing";

describe("<Landing />", () => {
  it("renders Landing component", () => {
    const { asFragment } = render(<Landing />);
    expect(asFragment()).toMatchSnapshot();
  });
});
