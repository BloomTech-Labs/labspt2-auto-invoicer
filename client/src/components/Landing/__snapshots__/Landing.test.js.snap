// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> renders Landing component 1`] = `
<DocumentFragment>
  <div
    class="landing"
  >
    <p
      class="landing_text"
    >
      Auto-Invoicer helps small buisnesses generate high quality PDF invoices and track them. Click Buy Now to sign up for 3 free credits to create an invoice. Buisnesses can buy further credits or pay a subscription for unlimited credits.
    </p>
    <button
      class="button"
      type="submit"
    >
      Buy Now
    </button>
  </div>
</DocumentFragment>
`;
